docs/references/express/overview.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Clerk Express.js SDK
+description: Learn how to integrate Express.js into your Clerk application.
+---
+
+# Clerk Express.js SDK
+
+## Set up Clerk Express.js
+
+<Steps>
+
+### Create a Clerk application
+
+You need to create a Clerk application in your Clerk Dashboard before you can set up Clerk Express.js. For more information, check out our [Set up your application](/docs/quickstarts/setup-clerk) guide.
```

Comment:
```suggestion
You need to create a Clerk application in your Clerk dashboard before you can set up `@clerk/express`. For more information, check out our ["setting up your application"](/docs/quickstarts/setup-clerk) guide.
```

Reply:
We capitalize Clerk Dashboard all throughout the docs. I am going to change this back here.

Also, we don't ever put a guide's title in quotes - and I actually think it's better to not include the title at all, incase it ever changes. Let's modify the text to not mention a guide's title name.

```suggestion
You need to create a Clerk application in the Clerk Dashboard before you can set up `@clerk/express`. For more information on setting up a Clerk application, check out the [dedicated guide](/docs/quickstarts/setup-clerk).
```